Formula/conmon.rb

```diff
@@ -0,0 +1,23 @@
+class Conmon < Formula
+  desc "OCI container runtime monitor"
+  homepage "https://github.com/containers/conmon"
+  url "https://github.com/containers/conmon/archive/refs/tags/v2.0.29.tar.gz"
+  sha256 "eb4d5e157671a61b88786e44094775194e30e1d0ad0b9d50035532ece78dbc28"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "glib"
+  depends_on "glibc" if Formula["glibc"].any_version_installed?
```

Comment:
Can you clarify why this is needed?  The `glibc` shared libraries should always be found by the linker, and I'm pretty sure our GCC spec files also contain the flags so that the `glibc` headers can also be found.

Reply:
I added this dependency because I see it in one of the dependent package on its GitHub readme. I can try remove this line and see whether it still compiles.

Reply:
Yes, you should be able to remove it.  We either use the host `glibc`, or install our own copy, but either way we shouldn't ever need to explicitly declare it as a dependency.

Reply:
Thanks for your clarification. It still compiles after removing the `glibc` dependency.